test/asset/cache_test.dart

```diff
@@ -4,14 +4,13 @@
 @TestOn('vm')
 import 'dart:async';
 
-import 'package:test/test.dart';
-
 import 'package:build/build.dart';
 import 'package:build/src/asset/cache.dart';
+import 'package:test/test.dart';
```

Comment:
This was intentional, https://www.dartlang.org/effective-dart/style/#prefer-placing-third-party-package-imports-before-other-imports